### ISSUE TITLE
Cannot open DB Error: java.lang.NumberFormatException: For input string: ""

### ISSUE THREAD

**Issue by csapinoso** [orientechnologies/orientdb#8281]
## Expected behavior  
Connect to DB 

## Actual behavior  
```
orientdb {server=remote:localhost/sierralabs_db}> connect remote:localhost/sierralabs_db root
Enter password:

Connecting to database [remote:localhost/sierralabs_db] with user 'root'...
Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database 'sierralabs_db'

Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database 'sierralabs_db'

Error: com.orientechnologies.orient.core.exception.ODatabaseException: Cannot open database url=plocal:/orientdb/databases/sierralabs_db

Error: java.lang.NumberFormatException: For input string: ""
```

## Steps to reproduce  
I don't have the exact steps but I can attach the [DB file](https://github.com/orientechnologies/orientdb/files/2024278/sierralabs_db.zip). Basically, I just created a new DB and created a few Vertices and Edges with inheritance. I started added some string properties on a couple of vertices, marking them are mandatory and not null, I got an error Error: java.lang.NumberFormatException: For input string: "". Then after that I couldn't add any more properties without getting the same error, though it seemed like the property is created if I do hard refresh of the page. I tried logging out of studio and restarting my docker instance, then tried to sign in again using studio but could not log on to the same database again, I guarantee that I have the right credentials since I can log on to other databases. I tried using the console and same thing happens (shown above)

**Comment by laa** [comment]
@csapinoso I suppose you incorrectly set empty max value or maybe Studio set this value for you. Sure those cases should be correctly handled. I have provided a quick and dirty fix so you can build develop branch and work with the database.  Assigned to @tglman .

**Comment by csapinoso** [comment]
That makes sense, I think before the issue started I was able to create a property called index of type Integer, I set the minimum value to zero because wanted only positive integers but did not specify a max value. Studio allowed it though, I supposed it should have been validated in the front end and backend if that would have caused an exception. In this case, it made my entire DB unusable. Is there a way to recover from this?
